### ISSUE TITLE
ContactsPlugin().getContacts() returns duplicates

### ISSUE THREAD

**Issue by korenzerah** [BaseflowIT/flutter-contacts-plugin#10]
## 🐛 Bug Report

Using the library as explained in the readme file, i'm getting duplicates of contacts (compared their ids too).

### Expected behavior
Get unique contacts by their id.
### Reproduction steps
Calling ContactsPlugin().getContacts().
### Configuration

**Version:** 0.0.3

**Platform:** 
- [ ] :iphone: iOS
- [X] :robot: Android

**Comment by WizKi9d** [comment]
Did you get a solution for this? 2 years later (almost exactly) and I'm now having the same issue. Some contacts are returned 2-3 times. Others only once. Very unusual.

**Comment by HemlockBane** [comment]
It's 2023, and the issue still persists
